src/java.desktop/macosx/classes/sun/lwawt/LWWindowPeer.java

```diff
@@ -521,7 +521,7 @@ public final boolean isTextured() {
     public final void setTextured(final boolean isTextured) {
         if (textured != isTextured) {
             textured = isTextured;
-            updateOpaque();
+            if (isVisible()) updateOpaque();
```

Comment:
What is the root cause of the problem? Why we do not change the surface if the peer is invisible? If the texture property will be changed while the peer is invisible who and when will call the "updateOpaque"?